presidio-analyzer/analyzer/nlp_engine/nlp_artifacts.py

```diff
@@ -33,3 +33,9 @@ def set_keywords(nlp_engine, lemmas, language):
         keywords = \
             [item for sublist in keywords for item in sublist]
         return keywords
+
+    def __repr__(self):
+        return str(self.__dict__)
+
+    def to_json(self):
+        return self.__repr__()
```

Comment:
Why go through two functions? 
```suggestion
        return str(self.__dict__)
```